scenarios/retail/README.md

```diff
@@ -0,0 +1,100 @@
+# Recommendation systems for Retail
+
+An increasing number of online companies are utilizing recommendation systems (RS) to increase user interaction and enrich shopping potential. Use cases of recommendation systems have been expanding rapidly across many aspects of eCommerce and online media over the last 4-5 years, and we expect this trend to continue.
+
+Companies across many different areas of enterprise are beginning to implement recommendation systems in an attempt to enhance their customer’s online purchasing experience, increase sales and retain customers. Business owners are recognizing potential in the fact that recommendation systems allow the collection of a huge amount of information relating to user’s behavior and their transactions within an enterprise. This information can then be systematically stored within user profiles to be used for future interactions.
+
+## Typical Business Scenarios in Recommendation Systems for Retail
+
+The most common scenarios companies use are:
+
+* Others you may like (also called similar items): The "Others you may like" recommendation predicts the next product that a user is most likely to engage with or purchase. The prediction is based on both the entire shopping or viewing history of the user and the candidate product's relevance to a current specified product.

```

Comment:
It might be better to introduce the simpler "Recommended For You" case first, then introduce this and note that it is similar to Recommended For You but also includes relevance to a specific product the user has just or is engaging with.  Also, I prefer "You might also like" vs. "Others you may like", but not sure which or others are more prevalent in industry.

Reply:
the examples here were taken as placeholder from this link https://cloud.google.com/recommendations-ai/docs/placements. 

When you say:

> It might be better to introduce the simpler "Recommended For You" case first, then introduce this and note that it is similar to Recommended For You but also includes relevance to a specific product the user has just or is engaging with.

*then introduce this* refers to Others you may like? so are you asking to swap the order between recommended for you and others you may like?

Reply:
yeah, when i first read it it took me a while to sort out what the difference between the two are. 
So I would introduce Recommended For You first then have the Others you may like with the following description:
```The "Others you may like" recommendation is similar to the "Recommended For You" as it predicts the next product a user is likely to engage with or purchase. In addition to considering the the entire shopping or viewing history of the user, the relevance of the current specified product to other items is used to recommend additional products.
```